config/version.go

```diff
@@ -135,6 +136,11 @@ func GetAlgorandVersion() string {
 	return currentVersion.String()
 }
 
+// GetAlgorandVersionAndBuild retrieves the current version and build
+func GetAlgorandVersionAndBuild() string {
+	return fmt.Sprintf("algod/%d.%d.%d (%s; commit=%s) %s(%s)", currentVersion.Major, currentVersion.Minor, currentVersion.BuildNumber, currentVersion.Channel, currentVersion.CommitHash, runtime.GOOS, runtime.GOARCH)
```

Comment:
IMO this is too much for metric label. Better model could probably be a multi label approach like {v=x.y.z, ch=channel, commit=commitHash, os=GOOS, arch=GOARCH}

Reply:
Perfect - this touches on the description in the PR re. whether to break it out. Current pattern was from https://github.com/prometheus/node_exporter/blob/master/collector/uname.go (nodename, which is very similar to the string above), however in hindsight uname information is opaque information, where in this case there is full control of the data source.

I will update.